### ISSUE TITLE
Mention of the "You're posting too fast" message

### ISSUE THREAD

**Issue by iyra** [minimaxir/hacker-news-undocumented#21]
I sometimes get an error message that says "You're posting too fast, please slow down.", and although I think this comes up for people who really do just post too fast, to my knowledge the mods can enable this as a flag on your account if they don't want you to post so much. From the [HN source](https://github.com/wting/hackernews/blob/master/news.arc#L1520) (which of course they don't run verbatim, but it's a starting point).

**Comment by minimaxir** [comment]
I believe the behavior changed recently: I get the message (even while logged in) just by clicking very fast.

**Comment by iyra** [comment]
Personally I noticed that I only started to get the message after being told off by HN mods; I think it's a coincidence that they use the same message for clicking too fast _and_ for posting too fast, since two actions must trigger it: making a few posts in the space of 5 or 10 (say) minutes _and_ clicking around very fast. It's plausible to me that they see you post in topics that attract a lot of flame posts, then put the flag on your account.

**Comment by cwardata** [comment]
Yes, moderators can enable this as a flag on an account, the user then cannot post links or flag comments.

**Comment by steakknife** [comment]
There's two sets of behaviors people are conflating. Throttle browsing and throttling posting.

**Comment by Wowfunhappy** [comment]
I think this message might be Karma related? I used to get it frequently when I was new the site, but never anymore, and I post no less quickly.

**Comment by shmerl** [comment]
I get this message when number of replies in a thread is above certain threshold. Then I can't post for around an hour or so. Not sure if this is universal, mods never said anything to me, so I assume it's a general behavior? Would be pretty nasty if it's not general, and there is no way to check if there is some individual throttling going on.

**Comment by jlinBE** [comment]
This message is linked to number of downvotes: if you post a heavily downvoted message (around -4?) while low-karma you can't post at all for a couple of hours.  This throttling can happen with as few as three replies in a single thread, posted over the course of half an hour, so it can't be all frequency based.

**Comment by jlinBE** [comment]
See :
https://hn.nuxtjs.org/item/15507821

Specifically this bit:

> dang 1018 days ago

>That's a sign that your account is rate-limited. We rate limit accounts when they repeatedly get involved in flamewars or regularly break the site guidelines or post lots of low-quality (for HN) comments. It's a crude tool but one of the few we have to prevent such things from overrunning the site.

The thread also has interesting discussions on other mod tools: shadowbanning, IP bans, and the like.

Apparently the way to un-rate-limit an account is to send in an email and mention that you will "stick to civil, substantive comments that scrupulously follow" https://news.ycombinator.com/newsguidelines.html.  Some amount of convincing prostration may be required, as a sort of high-level turing test.  There does not seem to be a way to do this while not giving away your email address (which was not required at signup).

The moderator mentioned something about actual, repeated, human-written warnings before rate-limiting, but this does not seem to be the case now (2020) - I'm guessing it's almost all automated and score based due to the high volume.

**Comment by chadbrewbaker** [comment]
Just got this message. Throw a timestamp if you want a wait so users know how long. Seemed way longer than 10 seconds - I typed the title then copy pasted the URI to make sure there weren't any typos.  Title: "GPL Jailbreak - Another Reflection on Trusting Trust" , URI http://chadbrewbaker.github.io/2020/12/10/gpl-jailbreak.html.

**Comment by alphapapa** [comment]
@chadbrewbaker Of course the HN mods can ban accounts (and you can see dang saying so at times).  But AFAIK rate-limiting and shadowbanning can happen without moderator intervention by having enough comments downvoted enough in a period of time, especially within a thread.

So what often happens in political discussions on HN is 1) a person, such as yourself, posts dissenting views in response to several messages, 2) those messages are heavily downvoted by a brigade of users, 3) the person's account is rate-limited or shadowbanned, 4) the brigade freely posts comments against the dissenter because, being many users, each only posting one or two messages, and being of the censorious brigade (the dissenters usually oppose censorship and don't downvote for disagreement), their comments don't get downvoted, so they don't get rate-limited.

The end result is, very predictably, a form of mob censorship.  And since HN approves of downvoting for disagreement, nothing is done to stop it.

Of course, dang is frequently seen saying that people on both sides of the political spectrum think that the site is biased against their views, replete with quotes of people saying so.  

Yet the observation remains that on HN, comments espousing views on one side of the spectrum are heavily downvoted and hidden much more than those on the other side.  It doesn't take much browsing with showdead=on to see that.

Then, on certain topics, one may perceive an army of mercenary downvoters, which is the only apparent explanation for why some very reasonable, insightful comments, that don't even express a political view (at least, with respect to American politics), are downvoted to obscurity.  These downvoters seem to be able to trigger the HN rate-limiting when they don't want certain ideas to have prominence.  In fact, that's how I discovered this GitHub issue, by following a link in a comment by this 10-year HN user: https://news.ycombinator.com/item?id=25463648

dang often says that he's concerned about protecting HN as a vessel.  But he seems less concerned about the truthfulness of the contents of the vessel.  So IMO, HN is less appropriate as a place for reasoned, free discussion, and more appropriate as a place to gauge the opinions of certain demographics.

**Comment by shmerl** [comment]
>  And since HN approves of downvoting for disagreement

It's a pretty bad practice, even if HN approves. Downvoting for disagreement is form of trolling.

**Comment by IThinkImOKAY** [comment]
HN is turning into r*ddit.

**Comment by CrazyPython** [comment]
It's also IP-based. I was using a SOCKS proxy to a personal Cloud VPS server. After posting two comments on a new account with 1 karma, posting another got me "You're posting too fast. Please slow down. Thanks." I turned the proxy off and I could post comments on that 1 karma account again.

**Comment by krionical** [comment]
If you point out the evident conflict of interest Stripe has as a payment processor against certain businesses which it also is trying to break into the market concerning, you will get this "please slow down" business at about three comments max.

**Comment by Stokestack** [comment]
There's no excuse for letting users waste their time typing out a question or comment and THEN telling them they can't post.

**Comment by dakoner** [comment]
I am getting this error message, but it's not going away.  It looks to me like the moderators locked my account from commenting, likely https://news.ycombinator.com/threads?id=dekhn&next=29896811#29899626
It appears to be permanent, so I'm going to the try "email and prostrate" suggestion above.

**Comment by Flova** [comment]
I get "You're posting too fast. Please slow down. Thanks." even tho I have never posted anything if I try to post my first post.

**Comment by sgammon** [comment]
> censoring Republican rhetoric over Biden's international financial dealings

crazy how they shadowbanned you. can't imagine why. anyway back to this github issue

**Comment by lostmsu** [comment]
I got a general warning from the moderator last week and hit the limit today for the first time. Found elsewhere that the limit is usually 5 posts in 24h.

Really annoying that one as I wrote a comment on a topic of intersection between electromagnetism and general relativity and it got eaten by the limit.

**Comment by Stokestack** [comment]
The best advice is not to waste your time on Hacker News. The site and many of its users treat others like shit.

**Comment by bennyschmidt** [comment]
> The best advice is not to waste your time on Hacker News. The site and many of its users treat others like shit.

Reddit, HackerNews, StackOverflow... why are vote up/down communities so unbelievably toxic? It's like people love publicly shaming others as much as they love publicly bragging about themselves - guess it covers both sides of the "I'm the smartest" mentality without having to prove anything (beyond cheap talk).

**Comment by bitcharmer** [comment]
@alphapapa put it better then I ever could. HN has been dang's fiefdom for a long while now; he's just another incarnation of a toxic reddit mod.
